draft-opsarea-rfc5706bis.md

```diff
@@ -564,10 +564,28 @@ This document does not describe interoperability requirements. As such, it does
    their mail server stopped functioning because of overload of the DNS
    cache resolver.
 
-   The impact on performance may also be noted -- increased delay or
-   jitter in real-time traffic applications, or increased response time
-   in client-server applications when encryption or filtering are
-   applied.
+The impact of New Protocols or Protocol Extensions, and the results
+of new OAM tools developed for the New Protocols or Protocol
+Extensions, must be considered with respect to the performance of
+traffic delivery and the availability of ongoing manageability. For
+example, it must be noted if the New Protocols or Protocol Extensions
+or the OAM tools cause increased delay or jitter in real-time traffic
+applications, or increased response time in client-server
+applications. Further, if the additional traffic caused by OAM tools
+and data collection could result in the management plane becoming
+swamped, then this must be called out, and suitable mechanisms to
+rate limit the OAM must be provided. Consider three examples: in
+Bidirectional Forwarding Detection for MPLS {{?RFC5884}} it is
+possible to configure very rapid BFD transmissions (of the order of
+3ms) on a very large number of parallel Label Switched Paths (LSPs)
+with the result that the management systems and end nodes may become
+overwhelmed -- this can be protected by limiting applying limits to
+the number of LSPs that may be tested at once; notifications or logs
+from systems (through YANG or other means) should be rate-limited so
+that they do not flood the receiving management station; the
+application of sophisticated encryption or filtering rules need to
+be considered in the light of the additional processing they may
+impose on the "fast path" for traffic.
```

Comment:
"fast path" or hardware path?  Again, maybe I'm being overly pedantic.

Reply:
"Fast path" is not clear or descriptive enough. Which path? Processing path. I recommend looking at how RFC 6192 treats this.